models/teamMember.js

```diff
@@ -0,0 +1,31 @@
+const Team = require('./team')
+const User = require('./user')
+
+module.exports = function (sequelize, DataTypes) {
+  const TeamMember = sequelize.define('TeamMember', {
+    id: {
+      type: DataTypes.INTEGER,
+      primaryKey: true,
+      AllowNull: false
+    },
+    team_id: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      references: {
+        model: 'Team',
+        key: 'id',
+      },
+    },
+    user_id: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      references: {
+        model: 'User',
+        key: 'id',
+      },
+    },
+    timestamps: true,
+  })
+  TeamMember.belongsTo(Team, { foreignKey: 'team_id' })
+  TeamMember.belongsTo(User, { foreignKey: 'user_id' })

```

Comment:
Please add a return statement as well
`return TeamMember`
its expected in index.js on line no 51. thats why your build is failing : |

Reply:
okay